src/containers/table/Table.js

```diff
@@ -187,6 +189,36 @@ class TableView extends React.Component {
     });
   }
 
+  clearTags() {
+    const selected = this.state.selectedTags;
+    let unSelected = this.state.unSelectedTags;
+    for (let i = 0; i < selected.length; i++) {
+      unSelected.push(selected[i]);
+    unSelected.sort((a, b) => {
+      return a.toLowerCase().localeCompare(b.toLowerCase());
+    });  
+    }
+    this.setState({
+      unSelectedTags: unSelected,
+      selectedTags: []
+    });
+  }
+
+  clearStages() {
+    const selected = this.state.selectedStages;
+    let unSelected = this.state.unSelectedStages;
+    for (let i = 0; i < selected.length; i++) {
+      unSelected.push(selected[i]);
+    }
+    unSelected.sort((a, b) => {
+      return a.toLowerCase().localeCompare(b.toLowerCase());
+    });
+    this.setState({
+      unSelectedStages: unSelected,
+      selectedStages: []
+    });
+  }
+
```

Comment:
use better option of defining extra variable with initial state, so you could use it here.